src/main/scripts/src/app/app.component.scss

```diff
@@ -11,7 +11,7 @@ header {
 }
 
 main {
-  width: 100vw;
-  padding: 0 5vw;
-  overflow-y: auto;
+  margin: 10px;
+  height: calc(100vh - 90px);
```

Comment:
I am not sure about this. Search box can grow vertically and so the header might be higher than `90px`.